### PR TITLE
Prepared for 2.2.7 release

### PR DESCRIPTION
**Pull request by dusktreader**
#### What
Bumped backend versions to release 2.2.7

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

File: backend/CHANGELOG.rst [hunk 1]
```diff
@@ -6,8 +6,12 @@ This file keeps track of all notable changes to license-manager-backend
 
 Unreleased
 ----------
+
+2.2.7 -- 2022-05-10
+-------------------
 * Update docker-compose to use postgresql instead of postgres
 * Added search and sort to list endpoints.
+* Skipped 2.2.6 to sync with backend
```
Comment (julianaklulo) [inline backend/CHANGELOG.rst]:
```suggestion
* Skipped 2.2.6 to sync with agent
```

File: backend/pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "license-manager-backend"
-version = "2.2.5"
+version = "2.2.6"
```
Comment (julianaklulo) [inline backend/pyproject.toml]:
```suggestion
version = "2.2.7"
```

**Review by julianaklulo — CHANGES_REQUESTED** [review]

File: backend/pyproject.toml (hunk 2 above)
Comment (dusktreader) [reply backend/pyproject.toml]: Fixed in 3cee980

File: backend/CHANGELOG.rst (hunk 1 above)
Comment (dusktreader) [reply backend/CHANGELOG.rst]: Fixed in 3cee980

**Review by julianaklulo — APPROVED** [review]

**Pull request merged**